actions/bellsoft-liberica-dependency/main.go

```diff
@@ -69,10 +69,14 @@ func main() {
 		versions[fmt.Sprintf("%d.%d.%d-%d", r.FeatureVersion, r.InterimVersion, r.UpdateVersion, r.BuildVersion)] = r.DownloadURL
 	}
 
-	versions.GetLatest(inputs).Write(os.Stdout)
+	if o, err := versions.GetLatest(inputs); err != nil {
+		panic(err)
+	} else {
+		o.Write(os.Stdout)
+	}
 }
 
-type Release struct {
+	type Release struct {
```

Comment:
This looks like it introduces a formatting error. Running `go fmt ./...` fixes this one and a couple other errors.

Reply:
I’ll go back for them.